core/src/main/resources/jenkins/model/Jenkins/load-statistics.jelly

```diff
@@ -26,9 +26,10 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
   <l:layout title="${it.displayName} ${%Load Statistics}">
     <st:include page="sidepanel.jelly" />
+    <l:breadcrumb href="" title="${%Load Statistics}"/>
```

Comment:
Why `href`? Jellydoc says it can be omitted.

Reply:
```suggestion
    <l:breadcrumb title="${%Load Statistics}"/>
```

Reply:
it was to make it a link, not that it was very useful, I've removed for now